AdyenActions/UI/View Controllers/Await/AwaitView.swift

```diff
@@ -99,23 +99,17 @@ internal final class AwaitView: UIView {
     
     /// :nodoc:
     private func configureConstraints() {
-        var guid = layoutMarginsGuide
-        
-        if #available(iOS 11, *) {
-            guid = safeAreaLayoutGuide
-        }
-        
         let constraints = [
             icon.topAnchor.constraint(equalTo: topAnchor, constant: 16),
             icon.centerXAnchor.constraint(equalTo: centerXAnchor),
             
             messageLabel.topAnchor.constraint(equalTo: icon.bottomAnchor, constant: 25),
-            messageLabel.leadingAnchor.constraint(equalTo: guid.leadingAnchor, constant: 16),
```

Comment:
why not `layoutMarginsGuide`? here and other lines

Reply:
No idea, I am simply removing  `if #available(iOS 11, *) { ... }`